dev-libs/libuv/libuv-1.9.1.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A new platform layer for Node"
+DESCRIPTION="libuv is a multi-platform support library with a focus on \
+asynchronous I/O. It was primarily developed for use by Node.js, but it's also \
+used by Luvit, Julia, pyuv, and others."
+HOMEPAGE="http://libuv.org/"
+COPYRIGHT="2009-2015 Ryan Dahl and others"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/libuv/libuv/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a6ca9f0648973d1463f46b495ce546ddcbe7cce2f04b32e802a15539e46c57ad"
+PATCHES="libuv-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libuv$secondaryArchSuffix = $portVersion
+	lib:libuv$secondaryArchSuffix = $portVersion = 1.0.0 compat >= 1
```

Comment:
Oops, you kept the **`= $portVersion`** while you should have removed it. The line should look like this:
```
	lib:libuv$secondaryArchSuffix = 1.0.0 compat >= 1
```